### DISCUSSION TITLE
Lesson 12: Upgrades - brownie.exceptions.VirtualMachineError: revert

### DISCUSSION THREAD

**Discussion by haraslub** [smartcontractkit/full-blockchain-solidity-course-py#914] [General]
Hello everyone,
I have a problem which is not related to this course but as I use the same (slightly adjusted) piece of code from this lecture, it may interested you. If this is not the case, I am sorry in advance. 

The problem is related to **Upgrade of the ERC1155** contract. I posted it at: https://ethereum.stackexchange.com/questions/120453/upgradable-proxy-to-erc1155
If anyone could help me, I would really appreciate it.
Thank you very much.

**Comment by cromewar** [comment]
Hello @haraslub well you have a tricky problem in hands; I'm willing to try to reproduce the error and find the bug but could you share a Github repo instead? So I can clone it and try to debug it?

**Reply by haraslub** [reply]
Sure, will do that and share it. Thank you very much, @cromewar !

**Reply by haraslub** [reply]
https://github.com/haraslub/erc1155-proxy @cromewar here we go, thank you again
